frog/imports/client/Wiki/components/TopNavbar/PrimaryButton.js

```diff
@@ -1,16 +1,17 @@
-//@flow
+// @flow
 
 import * as React from 'react';
 
 type PrimaryButtonPropsT = {
   title: string,
   active?: boolean,
   icon?: React.ComponentType<*>,
-  callback?: () => void
+  callback?: () => void,
+  i?: boolean
```

Comment:
this variable is not very descriptive, maybe rename as italic

Reply:
Made the changes